src/lib.rs

```diff
@@ -374,46 +374,60 @@ pub fn load_and_index_pois(
         (
             SELECT
                 geometry,
-                global_id_from_imposm(osm_id) as id,
-                st_x(st_transform(geometry, 4326)) as lon,
-                st_y(st_transform(geometry, 4326)) as lat,
+                global_id_from_imposm(osm_id) AS id,
+                st_x(st_transform(geometry, 4326)) AS lon,
+                st_y(st_transform(geometry, 4326)) AS lat,
                 poi_class(subclass, mapping_key) AS class,
                 name,
                 mapping_key,
                 subclass,
                 tags,
-                'osm_poi_point' as source
+                'osm_poi_point' AS source
             FROM osm_poi_point
                 WHERE name <> ''
             UNION ALL
             SELECT
                 geometry,
-                global_id_from_imposm(osm_id) as id,
-                st_x(st_transform(geometry, 4326)) as lon,
-                st_y(st_transform(geometry, 4326)) as lat,
+                global_id_from_imposm(osm_id) AS id,
+                st_x(st_transform(geometry, 4326)) AS lon,
+                st_y(st_transform(geometry, 4326)) AS lat,
                 poi_class(subclass, mapping_key) AS class,
                 name,
                 mapping_key,
                 subclass,
                 tags,
-                'osm_poi_polygon' as source
+                'osm_poi_polygon' AS source
             FROM osm_poi_polygon
                 WHERE name <> ''
             UNION ALL
             SELECT
                 geometry,
-                global_id_from_imposm(osm_id) as id,
-                st_x(st_transform(geometry, 4326)) as lon,
-                st_y(st_transform(geometry, 4326)) as lat,
+                global_id_from_imposm(osm_id) AS id,
+                st_x(st_transform(geometry, 4326)) AS lon,
+                st_y(st_transform(geometry, 4326)) AS lat,
                 'aerodrome' AS class,
                 name,
-                'aerodrome' as mapping_key,
-                'airport' as subclass,
+                'aerodrome' AS mapping_key,
+                'airport' AS subclass,
                 tags,
-                'osm_aerodrome_label_point' as source
+                'osm_aerodrome_label_point' AS source
             FROM osm_aerodrome_label_point
                 WHERE name <> ''
-        ) as unionall
+            UNION ALL
+            SELECT
+                geometry,
+                global_id_from_imposm(osm_id) AS id,
+                st_x(st_transform(geometry, 4326)) AS lon,
+                st_y(st_transform(geometry, 4326)) AS lat,
+                'city' AS class,
```

Comment:
Not sure if this is the correct naming... Same question for `mapping_key` and `subclass`.

Reply:
I think "mapping_key" is no longer used here, so it could also be removed from each subquery.

`hamlet` as subclass is fine. :heavy_check_mark: 

`city` as class looks too specific to me.  
We could use `place` (to be consistent with osm [`place=hamlet`](https://wiki.openstreetmap.org/wiki/Tag:place%3Dhamlet)). But this may be confusing too.  
What about `locality` ?